lib/Bio/EnsEMBL/DataCheck/Checks/CheckMethodLinkSpeciesSetNames.pm

```diff
@@ -0,0 +1,73 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckMethodLinkSpeciesSetNames;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckMethodLinkSpeciesSetNames',
+  DESCRIPTION    => 'Check for broken entries in method_link_species_set',
```

Comment:
```suggestion
  DESCRIPTION    => 'Check for consistency of names in method_link_species_set (and species_set_header)',
```

Reply:
I guess that change needs to be replicated in the index.json

Reply:
With the DC merge, will have to sort the `index.json` out anyway :+1: